config/locales/views/en.yml

```diff
@@ -2182,6 +2182,9 @@ en:
       submit: Change Username
       submit_landmark: Submit
       username_requirements: "%{minimum} to %{maximum} characters (A-Z, a-z, _, 0-9 only), no spaces, cannot begin or end with underscore (_)"
+    changed_username:
+      user:
+        not_different: Your new username must be different from your current username.
```

Comment:
In addition to the changes required for my other comment, this should go under https://github.com/otwcode/otwarchive/blob/ba22585f6d177a2ec2f9bc16e07a2e6f242807dd/config/locales/controllers/en.yml#L217

since it's in a controller, not a view.